e2e-polybft/framework/test-cluster.go

```diff
@@ -620,7 +651,7 @@ func (c *TestCluster) InitTestServer(t *testing.T,
 		}
 	}
 
-	srv := NewTestServer(t, c.Config, bridgeJSONRPC, func(config *TestServerConfig) {
+	srv := NewTestServer(t, c.Config, bridgeJSONRPC, byzantine, func(config *TestServerConfig) {
```

Comment:
Shouldn't byzantine parameter be part of `TestServerConfig`? I thought @Nemanja0x already mentioned it?

Reply:
I think the problem here is that `TestServerConfig` is instantiated inside of `NewTestServer()`.

Reply:
Yes, Nemanja already changed that. Byzantine is part of `TestServerConfig` and that config is created in `NewTestServer`. `t.Config` as an argument in `NewTestServer` is `ClusterConfig`.

Reply:
> Edit: But we have a callback, so no need to pass this param at all, we can set it through the callback.

What an unexpected plot twist :slightly_smiling_face: Yes made an oversight, but something was still suspicious :mag: 
So let's probably populate it via callback, if that is feasible and make sense...?